app/assets/stylesheets/app/card.scss

```diff
@@ -0,0 +1,139 @@
+.card {
+  border-radius: 2px;
+  background-color: $white;
+  font-family: $main-font;
+
+  &__title {
+    height: 48px;
+    font-weight: 300;
+    line-height: 48px;
+    text-align: center;
+    color: $secondary-color;
+  }
+
+  &__info {
+    font-size: 16px;
+    line-height: 24px;
+    text-align: center;
+    width: 850px;
+    margin: 0 auto;
+    color: $primary;
+    opacity: .64;
+  }
+
+  &-extended {
+    @extend .card;
+    justify-content: space-evenly;
```

Comment:
Properties should be ordered color, display, flex-direction, justify-content, margin, margin-top, min-height, width